### ISSUE TITLE
yolov3.conv.105 vs yolov3.conv.81vs darknet53.conv.74

### ISSUE THREAD

**Issue by takuhara** [AlexeyAB/darknet#1023]
I am trying to learn the 2 class problem.
One is person and the other is a minor class which is neither imagenet nor coco.

In yolov3, I know the following three pre-trained models,
Which do you think is the best in this case?
The reason for this is also?

 yolov3.conv.105
 yolov3.conv.81
 darknet53.conv.74

**Comment by AlexeyAB** [comment]
Try to use `yolov3.conv.81`

**Comment by baolinhu** [comment]
where can i get the yolov3.conv.81 or yolov3.conv.105？I only find darknet53.conv.74. thanks.

**Comment by takuhara** [comment]
no download link.see below.

https://github.com/AlexeyAB/darknet/commit/ba70801e982ada241ced55bedad2a411da1896c4#diff-2a86661590614a8bbb5321cf28e9ed97

**Comment by AlexeyAB** [comment]
@baolinhu 

Download file: https://pjreddie.com/media/files/yolov3.weights

How to get file `yolov3.conv.81` https://github.com/AlexeyAB/darknet/blob/eff487ba3626a39e135d13929117e04bc4cf5823/build/darknet/x64/partial.cmd#L21

**Comment by lvshuaigg** [comment]
What is the difference between yolov3.conv.105 and yolov3.conv.81.Thanks.

**Comment by AlexeyAB** [comment]
@lvshuaigg 
yolov3.conv.105 takes 105 layers from yolov3.weigths while yolov3.conv.81 takes 81 layers [0 - 80] - ` yolov3.conv.81` is correct pre-trained weights.
yolov3.conv.105 is incorrect - because layer-81 depends on number of classes, so higher layers will be shifted if number of classes will be changed.

**Comment by lvshuaigg** [comment]
Thank you very much. 
1. If I only train one class, what's the difference between yolov3.conv.81 and darknet53.conv.74? 
2. What is the difference if I train a lot of classes?

**Comment by AlexeyAB** [comment]
@lvshuaigg 
You can use any of them: yolov3.conv.81 (is Trained on MS COCO) or darknet53.conv.74 (is trained on Imagenet)

* `darknet53.conv.74` takes **74 layers** from `darknet53.weights`
* `yolov3.conv.81` takes **81 layers**  from `yolov3.weigths`

**Comment by kmsravindra** [comment]
@AlexeyAB , Can I use darknet53.conv.74 for 832 x 480 aspect ratio? Isn't Pascal VOC data 1:1 aspect images? Does using darknet53.conv.74 for 832 x 480 reduce accuracy?

**Comment by AlexeyAB** [comment]
@kmsravindra Yes, you can. darknet53 is trained with crop-data-augmentation (like jitter in the Yolo): https://github.com/AlexeyAB/darknet/blob/21a4ec9390b61c0baa7ef72e72e59fa143daba4c/cfg/darknet53.cfg#L13-L14
So it is slightly invariant to aspect ratio.

I think it will be better to use darknet53.conv.74 than not to use darknet53.conv.74 in your case.

**Comment by kmsravindra** [comment]
Thanks for your reply @AlexeyAB. I also meant to ask if you are aware of any other pre-trained weights for yolov3 that work better than darknet53.conv.74 for 832 x 480 aspect ratio.
